src/components/Calendar.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+const Calendar = (props) => {
+  const [startDate, setStartDate] = useState(new Date());
+  return (
+    <DatePicker selected={startDate} onChange={(date) => {
+      props.handleDate(date)
+      setStartDate(date)
+    }} />
+  );
+};
+export default Calendar;
```

Comment:
From now on, always leave an extra blank line at the end of a code file. It's a good practice.

Reply:
noted